wgsl/index.bs

```diff
@@ -8886,6 +8886,15 @@ member's type:
   Where |T| is the type of the |i|'th member of |S|.
 </p>
 
+If a structure member has the [=attribute/align=] attribute applied, the
+value [=shader-creation error|must=] be at least as large as the alignment of
```

Comment:
The note at the `align` attribute is expressed as "must be a positive multiple of ..." instead of just greater.
Shouldn't it be phrased the same way here?